data/interfaces.txt

```diff
@@ -18,6 +18,7 @@ color "bright" .75 .75 .75 0.
 color "medium" .5 .5 .5 0.
 color "dim" .25 .25 .25 0.
 color "faint" .1 .1 .1 0.
+color "extrafaint" .05 .05 .05 0.
```

Comment:
EIther two words "extra faint" (no other color definition smashes the words together like this), or choose a different word. How about this?
```suggestion
color "dark" .05 .05 .05 0.
```

Reply:
Hah, damn it. Was wracking my brain for a word darker than faint, and it was right there all along.